Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -309,6 +309,23 @@ PhysicalParticleContainer::AddGaussianBeam(Real x_m, Real y_m, Real z_m,
                   1, particle_w.dataPtr(),1);
 }
 
+void
+PhysicalParticleContainer::AddPlasmaFromFile(const std::string s_f)
+{
+#ifdef WARPX_USE_OPENPMD
+    openPMD::Series series = openPMD::Series(s_f, openPMD::AccessType::READ_ONLY);
+    amrex::Print() << "openPMD standard version " << series.openPMD() << "\n";
+
+    openPMD::Iteration& i = series.iterations[1];
+    amrex::Print()  << "File contains " << i.particles.size() << " specie(s):" << "\n";
+    for( auto const& ps : i.particles ) {
+        amrex::Print() << "\t" << ps.first << "\n";
+    amrex::Print()<<"WARNING: this is WIP, no particle has been injected!!";
+    }
```

Comment:
Oops sorry, actually I think my suggestion was wrong, there's no reason to put this inside the `for` loop.
```suggestion
    }
    amrex::Print()<<"WARNING: this is WIP, no particle has been injected!!";
```

Reply:
Ah yes yes, good point!